CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 Change Log
 ==========
 
+3.0.0
+-----
+
+* Upgraded to Django 2.2.
+* Dropped Support for Django 1.11 and versions below.
```

Comment:
I think since we never gave the previous x.x.x additions a release, we can just group those under your new 3.0.0 release.